app/keepers.go

```diff
@@ -161,6 +163,8 @@ type appKeepers struct {
 	SuperfluidKeeper     *superfluidkeeper.Keeper
 	GovKeeper            *govkeeper.Keeper
 	WasmKeeper           *wasm.Keeper
+	// Osmosis wasm query interface
+	OwasmKeeper owasm.ViewKeeper
```

Comment:
I wonder if we need to add this here?

I would propose just making this a local variable (that is added to wasmOpts) and not include it here as it is not a keeper, just a conversion wrapper.